medusa/name_parser/parser.py

```diff
@@ -36,12 +36,12 @@
 class NameParser(object):
     """Responsible to parse release names."""
 
-    def __init__(self, show=None, try_indexers=False, naming_pattern=False, parse_method=None,
+    def __init__(self, series=None, try_indexers=False, naming_pattern=False, parse_method=None,
```

Comment:
Same here, should be series_obj. If renamed consistently with the other renames.